front/src/common/Map/Layers/InfraObjectLayers/OperationalPoints.tsx

```diff
@@ -22,11 +20,9 @@ const OperationalPointsLayer = ({
   layerOrder,
   infraID,
   operationnalPointId,
-  overrideStore = false,
+  overrideStore,
```

Comment:
overrideStore was set to false by default, don't change that :+1: 
```suggestion
  overrideStore = false,
```